README.md

```diff
@@ -155,6 +155,9 @@ Here is an example of the tweaks needed to add `Pods` and `ConfigMaps` to the wa
    ```bash
    helm upgrade -n default --reuse-values -f charts/kubearchive/values.yaml [deployment name] charts/kubearchive --version 0.0.1
    ```
+   > **_NOTE:_**  A tweak is needed to this command for successfully deploying the api-server.
+   Check the section below.
```

Comment:
I think you should reference the name of the section you are referring to since it's not immediately after this

Reply:
Good point! Solved